dkim.js

```diff
@@ -173,12 +173,12 @@ class DKIMObject {
             }
         }
 
-        this.debug(this.identity + ': DKIM fields validated OK');
+        this.debug(`${this.identity}: DKIM fields validated OK`);
         this.debug([
-            this.identity + ':',
-            'a=' + this.fields.a,
-            'c=' + this.headercanon + '/' + this.bodycanon,
-            'h=' + this.signed_headers,
+            `${this.identity}:`,
+            `a=${this.fields.a}`,
+            `c=${this.headercanon}/${this.bodycanon}`,
+            `h=${this.signed_headers}`,
         ].join(' '));
```

Comment:
consider making this a single template string, instead of 4 array elements joined with a space.